starters/apps/basic/package.json

```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "__qwik__": {
     "priority": 2,
-    "displayName": "Basic App",
+    "displayName": "Basic App (Qwik City)",
```

Comment:
Maybe change the name to `Qwik City + Qwik` to remove any confusion and then we are good to merge?